### DISCUSSION TITLE
Unable to use "date" type in date_filter

### DISCUSSION THREAD

**Discussion by NiekFi** [developmentseed/tipg#144] [Q&A] [answered]
Hi,
we found that we are unable to use the our date columns to filter data on using the date_filter. It appears they are coded to search for timestamp & timestampz column types. Is there a particular reason for not including date column type? As a work around we'll be attempting to use the CQL filter to just filter on that date column.
Thanks :)

**Comment by vincentsarago** [comment]
I suppose we could add `date` as valid type, not sure why we didn't.

Maybe @bitner can answer 🙏

**Reply by bitner** [reply, marked as answer]
Simply an oversight

**Reply by NiekFi** [reply]
Haha no worries!

**Comment by vincentsarago** [comment]
@NiekFi it should be fine now with `0.5.3` 🥳
